relations.csv

```diff
@@ -4686,3 +4686,11 @@ HGNC,OPRD1,isa,FPLX,Opioid_receptor
 HGNC,OPRK1,isa,FPLX,Opioid_receptor
 HGNC,OPRL1,isa,FPLX,Opioid_receptor
 HGNC,OPRM1,isa,FPLX,Opioid_receptor
+HGNC,CCNH,isa,FPLX,CDK_activating_kinase_complex
+HGNC,CDK7,isa,FPLX,CDK_activating_kinase_complex
+HGNC,MNAT1,isa,FPLX,CDK_activating_kinase_complex
+HGNC,TIMM10,isa,FPLX,TIM22_complex
+HGNC,TIMM10B,isa,FPLX,TIM22_complex

```

Comment:
maybe part of would be a better relation? But I'm not familiar with TIM22 complex so you might know better

Reply:
It should absolutely be `partof`. These were imported by a script and I didn't notice that it defaulted to `isa`.